blackjack/game.py

```diff
@@ -0,0 +1,127 @@
+        self.dealer = Dealer()
+
+
+    def start_hand(self):
+        self.player.bank -= 10
+        self.deck = Deck()
+        self.deck.shuffle_deck()
+        self.player.take_a_hit(self.deck)
+        self.dealer.take_a_hit(self.deck)
+        self.player.take_a_hit(self.deck)
+        self.dealer.take_a_hit(self.deck)
+        self.interface.show_current(self.player, self.dealer)
+
+    def blackjack_check(self):
+        if self.dealer.blackjack():
+            if self.player.blackjack():
+                self.interface.show_final(self.player, self.dealer)
+                self.player.bank += 10
+                self.interface.you_push(self.player)
+                #self.clear_hands()
+                return False#Hand is over
+            else:
+                self.interface.show_final(self.player, self.dealer)
+                self.interface.you_lose(self.player)
+                #self.clear_hands()
+                return False#Hand is over
+        elif self.player.blackjack():
+            self.player.bank += 25
+            self.interface.blackjack(self.player)
+            #self.clear_hands()
+            return False#Hand is over
+
+    def player_turn(self):##Come back to this
+        while True:
+            if self.interface.player_choice(self.player, self.deck):
+                self.interface.show_current(self.player, self.dealer)
+                return True#if player stands
+            self.interface.show_current(self.player, self.dealer)
+            if self.player.hand_value() > 21:
+                self.interface.you_lose(self.player)
+                self.clear_hands()
+                return False#end hand
+
+
+    def dealer_turn(self):
+        while True:
+            self.interface.show_dealer_hand(self.dealer)
+            self.interface.show_dealer_total(self.dealer)
+            if self.dealer.hand_value() <= 16:
+                self.dealer.take_a_hit(self.deck)
+                return True
+            elif self.dealer.hand_value() >= 17:
+                return False
+
+
+    def win_loss_check(self):
+        self.interface.show_final(self.player, self.dealer)
+        if self.dealer.hand_value() > 21:
+            self.player.bank += 20
+            self.interface.you_win(self.player)
+            self.clear_hands()
+        elif self.dealer.hand_value() > self.player.hand_value():
+            self.interface.you_lose(self.player)
+            self.clear_hands()
+        elif self.dealer.hand_value() == self.player.hand_value():
+            self.player.bank += 10
+            self.interface.you_push(self.player)
+            self.clear_hands()
+        else:
+            self.player.bank += 20
+            self.interface.you_win(self.player)
+            self.clear_hands()
+
+    def check_bank(self, player):
+        if self.player.bank > 0:
+            print("Time to play a new hand")
+        else:
+            self.interface.no_money(self)
+            return sys.exit
```

Comment:
You don't need this line -- you're going to return anyway, and the game will end without the `sys.exit`. Also, you're not actually calling `sys.exit`, so you're returning a function.